platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMapOptions.java

```diff
@@ -217,7 +218,7 @@ public static MapboxMapOptions createFromAttributes(@NonNull Context context, @N
       Drawable compassDrawable = typedArray.getDrawable(
         R.styleable.mapbox_MapView_mapbox_uiCompassDrawable);
       if (compassDrawable == null) {
-        compassDrawable = ContextCompat.getDrawable(context, R.drawable.mapbox_compass_icon);
+        compassDrawable = ResourcesCompat.getDrawable(context.getResources(), R.drawable.mapbox_compass_icon, null);
```

Comment:
@tobrun 
It allows you to provide the `Theme` argument yourself.
Using `ContextCompat`:
> Starting in LOLLIPOP, the returned drawable will be styled for the specified Context's theme.
They're deprecating all methods in which you don't specify the `Theme`, so it seems the way to go 😉